cosim/black-parrot-example/htif/byteorder.h

```diff
@@ -0,0 +1,101 @@
+// See LICENSE for license details.
```

Comment:
Where's this code from? Should put in black-parrot-sdk?

Reply:
The PK part is going to go into perch. This part is not compiled into the program and runs on the host.